docker/Dockerfile-api

```diff
@@ -15,9 +15,11 @@ RUN dnf -y install \
     openssl-devel \
     python3-devel \
     python3-pip \
+    && dnf update -y \
```

Comment:
I tried today to install and make it work and i was missing package : `python3-mod_wsgi`. So maybe it is useful to add it

Reply:
This is interesting. I don't have a problem with `httpd`. My API is running correctly. Could you share how do you use dev env?

Reply:
Strange i cannot reproduce it now. You can ignore it.